README.md

```diff
@@ -53,8 +65,11 @@ The following simple HTML code illustrates the way to embed library view into an
         <script src="./node_modules/react/dist/react.js"></script>
         <script src="./node_modules/react-dom/dist/react-dom.js"></script>
 
-        <!-- The main library view component -->
-        <script src="./dist/bundle.js"></script>
+        <!-- The main library view compoment will be injected depending on build environment-->
+        <!-- start:librarie inject -->
+        <script src = './dist/librarie.min.js'></script>
+        <!-- end:librarie inject -->
+
```

Comment:
Can we restore these few lines? This is meant to be a reference document and therefore shouldn't include things like `inject` stuff.